config/ChangeLog

```diff
@@ -1,3 +1,7 @@
+2024-08-17 Ammar Almoris <ammaralmorsi@gmail.com>
```

Comment:
Please start Changelog entries with an empty line, that does help reducing merge conflicts for some reason (obviuously applies to the other Changelogs as well..